package.json

```diff
@@ -4,10 +4,11 @@
   "description": "GitHub label maker",
   "main": "index.js",
   "bin": {
-        "gh-label": "./index.js"
+    "gh-label": "./index.js"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "start": "node index"
```

Comment:
`start` script is unnecessary here because the purpose of this module is to use it as a globally available command once installed via NPM. Can you remove this ?